src/components/Contentful/sass/components/_contentful_grid.scss

```diff
@@ -84,3 +84,12 @@
 .contentful-grid.shaded + .contentful-grid {
   padding-top: map-get($spacers, 3);
 }
+
+.contentful-grid.black {
+  background-color: #333333;
+  margin-bottom: 0;
+
+  .prose {
+    color: #ffffff;
+  }
+}
```

Comment:
I think these colours for black grids should already be covered in _new_design_overrides.scss

Reply:
That's what I thought but I couldn't see them anywhere - I might have missed it though, I'll have a look

Reply:
You're correct - I think I hadn't tagged the page as new-design before and that's why I missed it. Removed.